src/features/landing/components/Icons/VilnaMenuIcon/VilnaMenuIcon.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+interface IVilnaMenuIconProps {
+  isActive: boolean;
+  style?: React.CSSProperties;
+}
+
+export default function VilnaMenuIcon({ isActive = false, style }: IVilnaMenuIconProps) {
+  if (isActive) {
```

Comment:
svg icon must be only like svg icon, this is bad approach to use two different svg icons in one react component